packages/now-static-build/src/index.ts

```diff
@@ -223,6 +224,20 @@ function getFramework(
   return framework;
 }
 
+async function fetchBinary(url: string, framework: string, version: string) {
+  const res = await fetch(url);
```

Comment:
Can we cache this? (maybe in memory?)

Reply:
But there is only one framework detected per deployment so the cache would never be hit if it was in memory 🤔 

It seems like its best to not cache in case a GitHub Release is deleted and then we would end up with a bad error message again.